code/modules/antagonists/gang/gang_themes.dm

```diff
@@ -0,0 +1,276 @@
+	gang_objectives = list(
+
+		/datum/antagonist/gang/yakuza = "Welcome to the station, new recruit. We here at Majima Construction are a legitimate enterprise, yadda yadda yadda.<br>\
+		Look, I'll cut to the chase. We're using this station as a money laundering operation. Here's what you and the rest of the schmucks need to do.<br>\
+		<B>Build something big, massive, and completely in the way of traffic on the station. Doesn't have to be anything in specific, just as long as it is expensive as fuck.</B>. \
+		And keep an eye out for anyone poking around our money. We suspect some auditors might be on the station as well.",
+
+		/datum/antagonist/gang/irs = "Congratulations, agent! You've been assigned to the Internal Revenue Service case against Nanotrasen and Majima Construction. \
+		We are proud of your success as an agent so far, and are excited to see what you can bring to the table today. We suspect that Nanotrasen and Majima Construction are engaging \
+		in some form of money laundering operation aboard this station.<br>\
+		<B>Investigate and stop any and all money laundering operations aboard the station, under the authority of the United States Government. If they do not comply, use force.</B>. \
+		Some station residents may try to tell you the United States doesn't exist anymore. They are incorrect. We simply went undercover after the Second American Civil War. The invisible one."
+	)
+
+/datum/gang_theme/wild_wasteland
+	name = "Wild, Wild Wasteland"
+	description = "Hey everybody, this is Three Dog, your friendly neighborhood disc jockey on 207.7! Today we got a shoutout to our man, the Captain on the Nanotrasen station in SSC territory! \
+	Our generous donator wanted us to say that, ahem, *crinkles paper*, 'Tunnel Snakes Rule'? Whatever that means, I'm sure it means a lot to the good captain! And now, we resume our \
+	10 hour marathon of Johnny Guitar, on repeat!"
+	involved_gangs = list(/datum/antagonist/gang/tmc, /datum/antagonist/gang/pg, /datum/antagonist/gang/tunnel_snakes)
+	gang_objectives = list(
+
+		/datum/antagonist/gang/tmc = "Welcome to the station, recruit. Here's how shit is gonna go down. \
+		We're the <B>ONLY</B> people who should have sick rides on this station. We're the Lost M.C., we own the streets. \
+		<B>Ensure that ONLY Lost M.C. members have access to any forms of vehicles, mechs, or wheeled transportation systems of any kind.</B> \
+		The Tunnel Snakes might take issue with this, remove them if you need to. And the Powder Gangers may damage our rides. Show them we mean business if they do.",
+
+		/datum/antagonist/gang/pg = "Alright buddy, we're in business now. It's time for us to strike back at Nanotrasen. \
+		They kept us, ALL of us in their damn debt slave labor prisons for years over minor debts and mistakes. \
+		<B>Ensure nobody else has to suffer under Nanotrasen's unlawful arrests by destroying the permabrig and the brig cells!</B> \
```

Comment:
might just wanna tell them to destroy the brig/armory/perma/sec in general, instead of just the cells.

Reply:
it's about sending a message